packages/react/src/components/card-notes/CardNotes.tsx

```diff
@@ -4,11 +4,29 @@ import TextArea from 'devextreme-react/text-area';
 import Toolbar, { Item } from 'devextreme-react/toolbar';
 import Button from 'devextreme-react/button';
 import { formatDate } from 'devextreme/localization';
+import Validator, { RequiredRule } from 'devextreme-react/validator';
+import ValidationGroup from 'devextreme-react/validation-group';
+import ScrollView from 'devextreme-react/scroll-view';
 
 import { Notes, Note } from '../../shared/types/card-notes';
 
 import './CardNotes.scss';
 
+let forceValidationBypass = true;
+const extendDefaultValidator = (e) => {
```

Comment:
If using this approach is the only option, consider extracting into a function that receives the ref to forceValidationBypass and returns the function, and reuse it to avoid duplication.
```
const createDefaultValidatorExtender = (bypassOptionRef) = (e) => { ....
```

Reply:
Fixed